### ISSUE TITLE
unneeded multiplication by 8 when parsing bitrate

### ISSUE THREAD

**Issue by mczerski** [tkem/mopidy-dleyna#70]
https://github.com/tkem/mopidy-dleyna/blob/7fb628e959e59537e550398c15c64ee6d373c506/mopidy_dleyna/translator.py#L98

I guess it should be not multiplied by 8 as bitrate is already in kbits/s

**Comment by tkem** [comment]
What makes you think so? What does your media server report?

**Comment by mczerski** [comment]
Well, the "Bitrate" string implies it is in bits not bytes. It not implies that is is in kilo bits though. I don't know the dlna protocol so I don't know if the protocol even defines if it is kbits or bits. But my media server is reporting in kbits.

**Comment by tkem** [comment]
> B.2.1.6 res@bitrate Namespace: DIDL-LiteProperty Data Type: xsd:unsignedInt Multi-Valued: NO
> Description:  The res@bitrate property  indicates  the bitrate  in bytes/second of  the  encoding of the resource.Note  that  there  exists  an  inconsistency  with  a res@bitrate property  name  and  its  value  being expressed in bytes/sec.

http://www.upnp.org/specs/av/UPnP-av-ContentDirectory-v4-Service.pdf

So your assumptions are based on one particular media server implementation, right? So it would be kind of helpful to mention which one...

**Comment by mczerski** [comment]
LOL, so they admit in the protocol definition that they are INCONSISTENT.
My NAS is Linksys WRT120AC router, which is using Twonky media server I believe.
So According to the specification You posted it should be * 8 / 1000 not only * 8. Mopidy requires the bitrate in kbits: [Track](https://docs.mopidy.com/en/latest/api/models/#mopidy.models.Track)

**Comment by tkem** [comment]
AFAIK, this hasn't been resolved, really: https://github.com/mopidy/mopidy/issues/1514

**Comment by mczerski** [comment]
ok, whatever. You can close the issue. I don't need the bitrate, it just seemed something is wrong to me while I was tinkering with the code.

**Comment by tkem** [comment]
yeah, this was never cleanly specified nor implemented, but nobody seemed to care too much about "bitrate"...
I'll leave this open, at least for future reference.
